queries.sql

```diff
@@ -6,3 +6,101 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT *FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+
+-- Update all animals to have species as "unspecified"
+UPDATE animals
+SET species = 'unspecified';
+
+-- Verify the change
+SELECT * FROM animals;
+
+-- Roll back the change
+ROLLBACK;
+
+-- Verify the species column reverted to its previous state
+SELECT * FROM animals;
+
+BEGIN;
+
+-- Update animals with names ending in "mon" to have species as "digimon"
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+-- Update animals without a species already set to have species as "pokemon"
+UPDATE animals
+SET species = 'pokemon'
+WHERE species = '';
+
```

Comment:
You have done very well with this transaction 👍 💪. I just wanted to draw your mind to something that may help you a lot along the line of writing standardized code. Since the species column is null (has nothing in it), it is always best to address it that way. Remember that saying a column is empty and saying a column contains an empty string is not the same thing and could even lead to some problems if you interchange these two states. I suggest that in your ```WHERE``` clause here, you use ```IS NULL``` instead 😉. The line may end up looking something like this: 

```sh
   WHERE species IS NULL;
```